spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -171,13 +170,6 @@ public String manifest(@PathVariable("name") String name, @PathVariable("version
 		return this.streamService.manifest(name, version);
 	}
 
-	@RequestMapping(path = "/history/{name}/{max}", method = RequestMethod.GET)
```

Comment:
why is the history entry point removed?

Reply:
sorry, this was my mistake. In Skipper's ReleaseController we got rid of the entire REST endpoint as it is served from the repository. But, here we use an explicit REST endpoint and it is required. will re-add.